### DISCUSSION TITLE
Version 2.0 Release

### DISCUSSION THREAD

**Discussion by marshallrobinson** [UGS-GIO/wetlands#44] [Show and tell]
Kudos to @wetlandsDiane, @jjhi11, @marthajensen for working on this major update!

It is now live on the website: https://geology.utah.gov/apps/wetlands/index.html

**Comment by wetlandsDiane** [comment]
Awesome! Thanks for working so hard on the review this week and last and to
Martha and Jay for getting all the changes made.

Is there no need for additional review at this point? I had planned to take
another look at it and to have someone on my team look at it as well, but
perhaps that is not necessary?

Thanks

On Tue, Mar 15, 2022 at 1:05 PM marshallrobinson ***@***.***>
wrote:

> Kudos to @wetlandsDiane <https://github.com/wetlandsDiane>, @jjhi11
> <https://github.com/jjhi11>, @marthajensen
> <https://github.com/marthajensen> for working on this major update!
>
> It is now live on the website:
> https://geology.utah.gov/apps/wetlands/index.html
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/UGS-GIO/wetlands/discussions/44>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AE3KRTXY46D2D3CTQJAD7U3VADNQ3ANCNFSM5QZUHJPQ>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 
Diane Menuz
Wetland Program Coordinator
Utah Geological Survey
1594 W. North Temple
P.O. Box 146100
Salt Lake City, UT 84114
801.538.7436
http://geology.utah.gov/resources/wetlands/

**Comment by marshallrobinson** [comment]
Re: need for additional review - 
Version 2.0 doesn't need additional review. 

Like always though, new issues can be opened in GitHub if/when bugs are found.
